### ISSUE TITLE
Main navigation features (zoom, pan) should be consistent between chart types

### ISSUE THREAD

**Issue by ebugden** [theia-ide/theia-trace-extension#511]
Basic navigation features (zoom with toolbar, select to zoom, pan) should be possible using the same buttons in both timeline and xy charts. Otherwise this is confusing for users.

**Comment by ebugden** [comment]
I think select to zoom is the only main navigation feature that is different between timeline and xy charts. @IbrahimFradj could you confirm or correct me on this?

**Comment by IbrahimFradj** [comment]
no, the panning is different, the panning in the xy-graph is faster than in the time-graph. The zoom between the xy-graph and the time-graph is now the same, so this problem has been solved.
